controlplane/kubeadm/config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
+namespace: kcp-system
```

Comment:
Given that we named the other one `capi-kubeadm-system`, how about being a little more consistent and explicit for these:

`capi-kubeadm-bootstrap-system`
`capi-kubeadm-controlplane-system`

Not sure if you strongly feel we need acronyms, but these names would be more descriptive.

cc @detiber

Reply:
I'm not strongly for/against acronyms, +1 to trying to maintain consistency, though.

Reply:
sure, these are fine

Reply:
one minor nit, `capi-kubeadm-control-plane-system` (control plane should be two words if we are to follow the kubernetes convention across their docs)